package.json

```diff
@@ -32,24 +32,24 @@
   },
   "homepage": "https://github.com/jackmellis/require-extension-hooks-vue#readme",
   "peerDependencies": {
-    "require-extension-hooks": "^0.3.0"
+    "require-extension-hooks": "0.3.3"
   },
   "dependencies": {
-    "@vue/component-compiler-utils": "^2.3.1",
-    "consolidate": "^0.15.0",
-    "postcss": "^7.0.0",
-    "postcss-modules-sync": "^1.0.0",
-    "source-map-support": "^0.5.8",
-    "vue-template-compiler": "^2.5.17"
+    "@vue/component-compiler-utils": "2.3.1",
```

Comment:
is there a reason these dependencies are all now `exact` versions? (without the ^) ?

Reply:
Nope that would be a force of habit. Let me re add the carrots back in.